uiowa/uiowa
# [Bug] Button color contrast fix

Resolves https://github.com/uiowa/uiowa/issues/7325

# How to test

- Review https://github.com/uiowa/uids/pull/932

```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /buttons-spans
```

- Compare against https://sandbox.prod.drupal.uiowa.edu/buttons-spans
- Confirm span icons render with correct color in layout builder preview (spans are rendered for layout builder preview, svgs are rendered for page view).
- Test other button pages https://sandbox.uiowa.ddev.site/page-buttons-and-icons
